tests/dynamics/test_synchronization.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+import xgi
+from xgi.exception import XGIError
+import numpy as np
+
+@pytest.mark.slow
```

Comment:
Question: How slow is slow? Would this test be much too different if we used a hypergraph with say 50 nodes instead of 100?

Reply:
It reduces it down from 8 seconds to 2 seconds for 10000 timesteps. We will either change the size of the hypergraph or reduce the timesteps to make it faster.